README.md

```diff
@@ -684,22 +685,47 @@ Special characters can be included:
 * Single Quote: `\'`
 * Unicode scalar: `\u{n}` where n is between one and eight hexadecimal digits
 
-#### Array Access Syntax
+The built-in methods and accessors for Strings are different in Swift 1.x, 2.1+, and 3.0, and so are not covered here.
+
+#### Array access syntax
 
 ```swift
-let example = [ "hi", "there", 23, true ]
-print("item at index 0: \(example[0])")
+// single item access, insertion and deletion
+var example = [ "hi", "there", 23, true ]
```

Comment:
If you were to run this code in an Xcode playground, it'll fail to compile and ask you to be explicit in declaring it as `[Any]`

Also, nitpick on code formatting / whitespace:

```swift
var example: [Any] = ["hi", "there", 23, true]
print("item at index 0: \(example[0])") // prints "hi"
example.insert("new item", at: 1) // "new item" is now the second in the list
example.remove(at: 0)
```